tests/test_dashboard_quadrants.py

```
@@ -51,3 +60,204 @@ async def test_jobs_preview_displays_jobs():
+@pytest.mark.asyncio
+async def test_config_preview_renders_sections():
+    """ConfigPreview should render config sections."""
+    config = create_mock_config(
+        {
+            "Watcher": {"check_interval": 30, "min_reward": 0.0},
+            "WebSocket": {"enable_websocket": True},
+        }
+    )
+
+    app = ConfigPreviewTestApp(config)
+    async with app.run_test() as pilot:
```

Ruff (0.14.14)

Local variable `pilot` is assigned to but never used

Remove assignment to unused variable `pilot`

(F841)